site2/docs/helm-deploy.md

```diff
@@ -330,19 +323,15 @@ cd pulsar-helm-chart
 
 The `prepare_helm_release` creates the following resources:
```

Comment:
This script was not maintained, so it was removed from the README  here https://github.com/apache/pulsar-helm-chart/pull/324. There are appropriate instructions in that PR that could be copied to this section. The current README in https://github.com/apache/pulsar-helm-chart should work, too.

Reply:
@michaeljmarshall thanks for the information. The [`Get started`](https://pulsar.apache.org/docs/next/getting-started-helm#step-1-install-pulsar-helm-chart) topic also introduces this script to create secrets. If we don't recommend users run it, what's the alternative used to create secrets?